code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -149,3 +149,15 @@
 	volume = 20
 	falloff_distance = 2
 	falloff_exponent = 5
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
remove all of these they're fugly and ruin documentation

Reply:
Done